### ISSUE TITLE
ILLink fails on when only .NET Core 3 is installed (Microsoft.AspNetCore.Blazor.Mono)

### ISSUE THREAD

**Issue by natemcmaster** [aspnet/AspNetCore#6726]
Microsoft.AspNetCore.Blazor.Mono 0.8.0-preview1-20181126.1 contains a tool illink.dll which only runs on .NET Core 2.x. This means builds will fail on machines which only have .NET Core 3.x SDKs will fail.

```
2019-01-15T23:23:03.2879513Z   It was not possible to find any compatible framework version
2019-01-15T23:23:03.2881025Z   The specified framework 'Microsoft.NETCore.App', version '2.0.0' was not found.
2019-01-15T23:23:03.2886552Z     - Check application dependencies and target a framework version installed at:
2019-01-15T23:23:03.2892068Z         F:\vsagent\11\.dotnet\x64\
2019-01-15T23:23:03.2896169Z     - Installing .NET Core prerequisites might help resolve this problem:
2019-01-15T23:23:03.2896890Z         https://go.microsoft.com/fwlink/?LinkID=798306&clcid=0x409
2019-01-15T23:23:03.2897099Z     - The .NET Core framework and SDK can be installed from:
2019-01-15T23:23:03.2898295Z         https://aka.ms/dotnet-download
2019-01-15T23:23:03.2898690Z     - The following versions are installed:
2019-01-15T23:23:03.2899551Z         3.0.0-preview-27113-06 at [F:\vsagent\11\.dotnet\x64\shared\Microsoft.NETCore.App]
2019-01-15T23:23:03.2904079Z         3.0.0-preview-27219-3 at [F:\vsagent\11\.dotnet\x64\shared\Microsoft.NETCore.App]
2019-01-15T23:23:03.3300512Z F:\vsagent\11\s\src\Components\Blazor\Build\src\targets\Blazor.MonoRuntime.targets(439,5): error MSB3073: The command "dotnet "F:\vsagent\11\s\.nuget\packages\microsoft.aspnetcore.blazor.mono\0.8.0-preview1-20181126.1\build\netstandard1.0\../../tools/illink/illink.dll" -l none --verbose --strip-security true --exclude-feature com --exclude-feature sre -v false -c link -u link -b true -d "F:\vsagent\11\s\.nuget\packages\microsoft.aspnetcore.blazor.mono\0.8.0-preview1-20181126.1\build\netstandard1.0\../../tools/mono/bcl/" -d "F:\vsagent\11\s\.nuget\packages\microsoft.aspnetcore.blazor.mono\0.8.0-preview1-20181126.1\build\netstandard1.0\../../tools/mono/bcl/Facades/" -o "F:\vsagent\11\s\src\Components\Blazor\testassets\StandaloneApp\obj\Release\netstandard2.0\blazor\linker/" -x "F:\vsagent\11\s\src\Components\Blazor\Build\src\targets\BuiltInBclLinkerDescriptor.xml" -x "F:\vsagent\11\s\src\Components\Blazor\testassets\StandaloneApp\obj\Release\netstandard2.0\blazor\linker.descriptor.xml" -a "F:\vsagent\11\s\.nuget\packages\microsoft.extensions.dependencyinjection\3.0.0-preview.19059.4\lib\netstandard2.0\Microsoft.Extensions.DependencyInjection.dll" -a "F:\vsagent\11\s\.nuget\packages\microsoft.extensions.dependencyinjection.abstractions\3.0.0-preview.19059.4\lib\netstandard2.0\Microsoft.Extensions.DependencyInjection.Abstractions.dll" -a "F:\vsagent\11\s\.nuget\packages\microsoft.jsinterop\0.8.0-preview1-20181126.4\lib\netstandard2.0\Microsoft.JSInterop.dll" -a "F:\vsagent\11\s\.nuget\packages\mono.webassembly.interop\0.8.0-preview1-20181126.4\lib\netstandard2.0\Mono.WebAssembly.Interop.dll" -a "F:\vsagent\11\s\src\Components\Blazor\Blazor\src\bin\Release\netstandard2.0\Microsoft.AspNetCore.Blazor.dll" -a "F:\vsagent\11\s\src\Components\Browser\src\bin\Release\netstandard2.0\Microsoft.AspNetCore.Components.Browser.dll" -a "F:\vsagent\11\s\src\Components\Components\src\bin\Release\netstandard2.0\Microsoft.AspNetCore.Components.dll" -a "F:\vsagent\11\s\src\Components\Blazor\testassets\StandaloneApp\obj\Release\netstandard2.0\StandaloneApp.dll"" exited with code -2147450730. [F:\vsagent\11\s\src\Components\Blazor\testassets\StandaloneApp\StandaloneApp.csproj]

**Comment by RemiBou** [comment]
I have a full repro [here]( https://github.com/RemiBou/Toss.Blazor/tree/raven-db) type "docker build -t toss .", I'll try to fix it by manually installing net core 2

**Comment by SteveSandersonMS** [comment]
This is fixed in #7792, though we may choose not to merge that in time for Preview 3 (because it causes an app size regression). So the netcore2.0 requirement may remain for Preview 3.

**Comment by SteveSandersonMS** [comment]
Will be fixed with #8782
